src/main/java/mate/academy/model/Driver.java

```diff
@@ -79,6 +79,8 @@ public String toString() {
                 + "id=" + id
                 + ", name='" + name + '\''
                 + ", licenseNumber='" + licenseNumber + '\''
+                + ", login='" + login + '\''
+                + ", password='" + password + '\''
```

Comment:
I wrote about equals and hashcode, not about toString()

Reply:
Oh sorry. ^__^ . My apologises. Refactored.